### DISCUSSION TITLE
"SQL query statistic" is not displayed in the report from remote server

### DISCUSSION THREAD

**Discussion by ArtObr** [zubkov-andrei/pg_profile#85] [Q&A] [answered]
Здравствуйте. Спасибо за отличный инструмент.
Я столкнулся с проблемой, надеюсь подскажете как её решить.

Я развернул отдельный чистый сервер psql, накатил pg_profile. На самом себе (local) выгрузка отрабатывает отлично.

Есть целевой сервер, статистика с которого интересна. Но при выгрузке get_report с этого сервера он пропускает самый интересный раздел - SQL query statistic (Так же пропускается раздел Statement statistics by database)
![image](https://github.com/zubkov-andrei/pg_profile/assets/25934159/1ca1c13c-70f6-47c6-89c8-11eecc6144e5)

Пробовал выгружать через psql -Aqtc, результат такой же.

**Comment by zubkov-andrei** [comment, marked as answer]
Здравствуйте!
Отсутствие в отчёте раздела по статистикам исполнения SQL-запросов наиболее вероятно связано с недоступностью расширения pg_stat_statements на целевом сервере. Убедитесть в том, что это расширение прописано в параметре shared_preload_libraries, а также в том, что расширение установлено на целевом сервере (CREATE EXTENSION).
В какой схеме будет установлено расширение не важно, главное чтобы оно было установлено в той базе данных, которая прописана в строке подключения.

**Reply by ArtObr** [reply]
Да, я забыл прописать в shared_preload_libraries. Всё заработало.
Огромное спасибо за оперативный ответ!
